### DISCUSSION TITLE
how to use neurokit on MIT-BIH arrhythmia database?

### DISCUSSION THREAD

**Discussion by nein-ala** [neuropsychology/NeuroKit#485] [Q&A] [answered]
hey I'm working on detecting life-threatening cardiac arrhythmias using MIT-BIH database . I need to do ecg segmentation and peak detection and heart beat detection on this database but i cant figure out how to do these actions with neurokit . somehow I feel like this package do not support the data format that I'm using . do you have any ideas how can I implement what I need to do ? 
this is a link to download the database : https://physionet.org/static/published-projects/mitdb/mit-bih-arrhythmia-database-1.0.0.zip

**Comment by zen-juen** [comment, marked as answer]
Hi @nein-ala sorry for the late response, we must have missed this by accident!

Regarding the MIT-BIH database, neurokit has a script for formatting the database here: https://github.com/neuropsychology/NeuroKit/blob/master/data/mit_arrhythmia/download_mit_arrhythmia.py
The instructions on how to do this are contained inside as well. If you haven't already, you can try this out and let us know if it works!

**Comment by nein-ala** [comment]
hi , thanks for your response . I used another method but I will try to use
the link for formatting the database for my work too and I'll let you know
if I succeed.
